Tests/Controller/ActionControllerTest.php

```diff
@@ -330,6 +331,21 @@ public function testActionTest()
         $this->assertEquals('{"test":"test"}', $response->getContent());
     }
 
+    public function testActionReadOnly()
+    {
+        $request = $this->prepareRequest('test');
+        $container = self::$container;
+        $container->set("syrup.storage_api", $this->getStorageServiceStubDcaPython());
```

Comment:
Nerozumiem, ako moze tento test prejst (vyhodit tu vynimku), ked tu nenastavujes tu roli a vlastne je to uplne stejny test ako ten pred nim ale jeden skonci success a tento vyhodi tu readonly vyjimku? Nieco mi unika :)

Reply:
No tak to nerozumis uplne spravne, ten test vyjimku nevyhodi protoze se nespousti a krome toho vubec nefunguje. Vubec to tam nemelo byt, ty sync akce bych si rozmyslel, protoze kdyz je zakazem, tak to v UI bude delat celkem bordel (treba DB extractory, kde se volaji automaticky) a hlavne pres sync akce by celkem nemelo jit nic zmenit.

Hodil jsem to cely dohaje